Client/Frontend/Fakespot/Views/FakespotOptInCardView.swift

```diff
@@ -0,0 +1,359 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+import Shared
+import ComponentLibrary
+
+// MARK: View Model
+
+struct FakespotOptInCardViewModel {
+    private let tabManager: TabManager
+    private let prefs: Prefs
+    let cardA11yId: String = AccessibilityIdentifiers.Shopping.OptInCard.card
+    var productSitename: String?
+    // MARK: Labels
+    let headerTitleLabel: String = .Shopping.OptInCardHeaderTitle
+    let headerLabelA11yId: String = AccessibilityIdentifiers.Shopping.OptInCard.headerTitle
+    let bodyFirstParagraphLabel: String = .Shopping.OptInCardCopy
+    let bodyFirstParagraphA11yId: String = AccessibilityIdentifiers.Shopping.OptInCard.optInCopy
+    let disclaimerTextLabel: String = .Shopping.OptInCardDisclaimerText
+    let disclaimerTextLabelA11yId: String = AccessibilityIdentifiers.Shopping.OptInCard.disclaimerText
+    // MARK: Buttons
+    let learnMoreButtonTitle: String = .Shopping.OptInCardLearnMoreButtonTitle
+    let learnMoreButtonTitleA11yId: String = AccessibilityIdentifiers.Shopping.OptInCard.learnMoreButtonTitle
+    let termsOfUseButtonTitle: String = .Shopping.OptInCardTermsOfUse
+    let termsOfUseButtonTitleA11yId: String = AccessibilityIdentifiers.Shopping.OptInCard.termsOfUseButtonTitle
+    let privacyPolicyButtonTitle: String = .Shopping.OptInCardPrivacyPolicy
+    let privacyPolicyButtonTitleA11yId: String = AccessibilityIdentifiers.Shopping.OptInCard.privacyPolicyButtonTitle
+    let mainButtonTitle: String = .Shopping.OptInCardMainButtonTitle
+    let mainButtonTitleA11yId: String = AccessibilityIdentifiers.Shopping.OptInCard.mainButtonTitle
+    let secondaryButtonTitle: String = .Shopping.OptInCardSecondaryButtonTitle
+    let secondaryButtonTitleA11yId: String = AccessibilityIdentifiers.Shopping.OptInCard.secondaryButtonTitle
+    // MARK: Actions
+    func onTapLearnMore() {
+        TelemetryWrapper.recordEvent(category: .action,
+                                     method: .tap,
+                                     object: .shoppingLearnMoreButton)
+//        tabManager.addTabsForURLs([], zombie: false, shouldSelectTab: true) // no urls yet, will be added in another task
```

Comment:
Do you have the task number? 
```suggestion
//        tabManager.addTabsForURLs([], zombie: false, shouldSelectTab: true) // no urls yet, will be added in  FXIOS-1111
```

Reply:
the plan was to have that functionality in this PR but we don't have the links so the task isn't created yet

Reply:
Ok can you create it? There's a lot of moving parts in this project. To ensure nothing slips it's better to have a task

Reply:
[done](https://mozilla-hub.atlassian.net/browse/FXIOS-7384)